test/verify/check-system-info

```diff
@@ -281,12 +284,18 @@ class TestSystemInfo(MachineCase):
         self.assertIn("Mon Jun  4 06:34:", m.execute("date"))
         self.assertIn("EEST 2018\n", m.execute("date"))
 
-    def testTimeServers(self):
+    @skipImage("timesyncd not available", "rhel-9-2", "rhel-8-8", "centos-8-stream")
```

Comment:
centos-9-stream as well -- not sure if we run this test in gating/packit (you'll get a red test -- if not, feel free to ignore)

Reply:
Oh, why are we not testing centos-9-stream as well?

Reply:
Somewhere in between "redundant with RHEL 9" and "most tests already run in packit" and "forgot about it", I suppose :grin: